spec/lib/tasks/schools_rake_spec.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+describe "schools:move_patients" do
+  def capture_rake_task_output(task_name, *args)
+    stdout = StringIO.new
+    $stdout = stdout
+    Rake::Task[task_name].invoke(*args)
+    $stdout = STDOUT
+    Rake.application[task_name].reenable
+    stdout.string
+  end
```

Comment:
It doesn't look like this output is being tested anywhere, so I think we can simplify this and instead call `Rake::Task["schools:move_patients"].invoke` directly, similar to https://github.com/nhsuk/manage-vaccinations-in-schools/blob/24f2fa48d3003e0977c47d37ddb0ae7edbf1222b/spec/lib/tasks/status_update_spec.rb#L5